cos-fleetshard-operator-debezium/src/main/java/org/bf2/cos/fleetshard/operator/debezium/model/AbstractApicurioConverter.java

```diff
@@ -0,0 +1,36 @@
+package org.bf2.cos.fleetshard.operator.debezium.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.bf2.cos.fleetshard.api.ManagedConnector;
+import org.bf2.cos.fleetshard.api.SchemaRegistrySpec;
+import org.bf2.cos.fleetshard.api.ServiceAccountSpec;
+import org.bf2.cos.fleetshard.operator.debezium.DebeziumConstants;
+
+public abstract class AbstractApicurioConverter implements KafkaConnectConverter {
+
+    public static final String APICURIO_AUTH_SERVICE_URL = "https://identity.api.openshift.com/auth";
+
+    @Override
+    public Map<String, String> getAdditionalConfig(ManagedConnector config, ServiceAccountSpec serviceAccountSpec) {
+        Map<String, String> additionalConfig = new HashMap<>();
+        additionalConfig.put("apicurio.auth.service.url", APICURIO_AUTH_SERVICE_URL);
+        additionalConfig.put("apicurio.auth.realm", "rhoas");
```

Comment:
Are those property needed ?  
I recall I did a demo with camel and they were not required

Reply:
Was your demo using OAUTH with RHOSR using the Kafka Connect converters? I think this is needed for them, yes!

Reply:
I was using the serdes from apicurio
If they are needed, IMHO, hey should not be hard-coded  but provided by a configuration option

Reply:
wdym with "provided by a configuration option? They are conditional properties that gets added for either the key or value converter dependent of the selected `data_shape`. So this is the case!

Reply:
I mean that the values from `apicurio.auth.service.url` and the `apicurio.auth.realm` should not be hard-coded but they should be retrieved from a secret.

Reply:
Thx @lburgazzoli! Now I gotcha!
I filed MGDCTRS-584](https://issues.redhat.com/browse/MGDCTRS-584) until we know values for different environments and it involves deployment of such secrets too.

Reply:
Can we add this later because this PR waits a while now and @ani-sha needs this for her work because it enhanced the cucumber tests that she needs for her work on the pull secret and we need the pull secret asap!